### PR TITLE
My overview topic_RS

### PR DESCRIPTION
**Pull request by radhikasundararaman24**
Please review and suggest changes.

### CODE REVIEW

File: docs/overview_rs.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+---
+
+# Overview_RS
+
+The To-Do service is a cloud-based platform that offers subscribers a task list 
+where they can post tasks and receive timely reminders. Our API service emulates the REST
```
Comment (rbwatson) [inline docs/overview_rs.md]:
```suggestion
where they can post tasks and receive timely reminders. Our service provides the REST 
```

The Service provides the API for other applications and services to use to access the features of the service.

File: docs/to-do-service_img.svg
Comment (rbwatson) [inline docs/to-do-service_img.svg:1]: Love the graphic!

**Review by rbwatson — CHANGES_REQUESTED** [review]
Great start!
When you update this, be sure to add a link to it from the "default" overview topic (`docs/index.md`).

**Comment by radhikasundararaman24** [comment]
PR - #89

**Pull request closed without merging**